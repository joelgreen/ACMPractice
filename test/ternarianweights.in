100
1
10
100
1000
10000
100000
1000000
10000000
100000000
1000000000
3
9
27
81
243
729
2187
6561
19683
59049
177147
531441
1594323
4782969
14348907
43046721
129140163
387420489
2
8
26
80
242
728
2186
6560
19682
59048
177146
531440
1594322
4782968
14348906
43046720
129140162
387420488
1081846
7112622
9655877
5324969
5643485
1691387
2278794
4019967
1595435
3856568
6224998
8069499
477364
2795058
5679784
535852
2244000
3698377
442703
7555638
341894
9922175
5722760
6637136
5695456
7998359
6585776
7103006
6028916
401057
8524601
2835281
8934246
9182201
1200523
8173667
2247668
1897450
7970279
4684
9503010
1211282
6672394
8999546
7450097
4203629
2375495
5829243
8506985
6463985
8403659
7468097
3212854
5205852
