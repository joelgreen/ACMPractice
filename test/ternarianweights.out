left pan:
right pan: 1

left pan:
right pan: 9 1

left pan: 9
right pan: 81 27 1

left pan:
right pan: 729 243 27 1

left pan: 6561 2187 729 243
right pan: 19683 27 9 1

left pan: 59049 19683 729 81 27 9
right pan: 177147 2187 243 1

left pan: 531441 59049 6561 243 27
right pan: 1594323 2187 729 81 1

left pan: 4782969 177147 27
right pan: 14348907 531441 59049 19683 729 243 81 9 1

left pan: 43046721 531441 59049 19683 6561 2187 729 81 9
right pan: 129140163 14348907 177147 243 1

left pan: 129140163 43046721 4782969 177147 2187
right pan: 1162261467 14348907 531441 6561 729 81 1

left pan:
right pan: 3

left pan:
right pan: 9

left pan:
right pan: 27

left pan:
right pan: 81

left pan:
right pan: 243

left pan:
right pan: 729

left pan:
right pan: 2187

left pan:
right pan: 6561

left pan:
right pan: 19683

left pan:
right pan: 59049

left pan:
right pan: 177147

left pan:
right pan: 531441

left pan:
right pan: 1594323

left pan:
right pan: 4782969

left pan:
right pan: 14348907

left pan:
right pan: 43046721

left pan:
right pan: 129140163

left pan:
right pan: 387420489

left pan: 1
right pan: 3

left pan: 1
right pan: 9

left pan: 1
right pan: 27

left pan: 1
right pan: 81

left pan: 1
right pan: 243

left pan: 1
right pan: 729

left pan: 1
right pan: 2187

left pan: 1
right pan: 6561

left pan: 1
right pan: 19683

left pan: 1
right pan: 59049

left pan: 1
right pan: 177147

left pan: 1
right pan: 531441

left pan: 1
right pan: 1594323

left pan: 1
right pan: 4782969

left pan: 1
right pan: 14348907

left pan: 1
right pan: 43046721

left pan: 1
right pan: 129140163

left pan: 1
right pan: 387420489

left pan: 531441 729
right pan: 1594323 19683 9 1

left pan: 243
right pan: 4782969 1594323 531441 177147 19683 6561 729 9 3

left pan: 4782969 59049 19683 6561 2187 1
right pan: 14348907 177147 243 27 3

left pan: 6561 2187 729 1
right pan: 4782969 531441 19683 243 81 27 3

left pan: 531441 177147 19683 6561 27 1
right pan: 4782969 1594323 729 243 81

left pan: 59049 19683 2187 1
right pan: 1594323 177147 729 81 27

left pan: 19683 6561 81 9
right pan: 1594323 531441 177147 2187 27 3

left pan: 531441 177147 59049 2187 9
right pan: 4782969 6561 243 27

left pan: 729 243 81 27 3 1
right pan: 1594323 2187 9

left pan: 1594323 59049 2187 81 3 1
right pan: 4782969 531441 177147 19683 729 243

left pan: 177147 2187 27
right pan: 4782969 1594323 19683 6561 729 81 9 3 1

left pan: 4782969 1594323 59049 19683 729 81
right pan: 14348907 177147 243 27 9

left pan: 59049 2187 243
right pan: 531441 6561 729 81 27 3 1

left pan: 1594323 531441 59049 9
right pan: 4782969 177147 19683 81

left pan: 531441 177147 6561 2187 81 27
right pan: 4782969 1594323 19683 243 9 1

left pan: 2187
right pan: 531441 6561 27 9 1

left pan: 59049 81 27
right pan: 1594323 531441 177147 243 3

left pan: 1594323 19683 2187 81 3
right pan: 4782969 531441 243 1

left pan: 177147 81 1
right pan: 531441 59049 19683 6561 2187 729 243 27 9 3

left pan: 4782969 1594323 531441 59049 2187 729
right pan: 14348907 177147 243 27 9 3

left pan: 177147 19683 9 1
right pan: 531441 6561 729 3

left pan: 4782969 177147 243 1
right pan: 14348907 531441 2187

left pan: 531441 177147 6561 729 1
right pan: 4782969 1594323 59049 2187 81 27 3

left pan: 2187 729 3 1
right pan: 4782969 1594323 177147 59049 19683 6561 243 81

left pan: 531441 177147 243 9
right pan: 4782969 1594323 19683 6561 729 27 3 1

left pan: 4782969 1594323 243 9 3 1
right pan: 14348907 19683 6561 729 27

left pan: 19683 6561 2187 9 1
right pan: 4782969 1594323 177147 59049 729

left pan: 2187 729 1
right pan: 4782969 1594323 531441 177147 19683 243 81 27 9

left pan: 531441 729 3 1
right pan: 4782969 1594323 177147 6561 81 9

left pan: 177147 19683 1
right pan: 531441 59049 6561 729 81 27

left pan: 4782969 1594323 243 81 1
right pan: 14348907 531441 19683 2187

left pan: 1594323 531441 81 1
right pan: 4782969 177147 729 243 27 9 3

left pan: 4782969 531441 177147 2187
right pan: 14348907 59049 19683 243 81 27

left pan: 4782969 531441 19683 6561 2187 729 243 27 9 3 1
right pan: 14348907 177147

left pan: 531441 59049 243 9
right pan: 1594323 177147 19683 81 27 3 1

left pan: 4782969 1594323 2187 1
right pan: 14348907 177147 19683 6561 729 81 27 9 3

left pan: 59049 2187 729 81 1
right pan: 1594323 531441 177147 6561 243

left pan: 177147 59049 729 243 3
right pan: 1594323 531441 6561 2187 81 27 1

left pan: 4782969 1594323 2187 81 27 9 3 1
right pan: 14348907 729 243

left pan: 2187 27
right pan: 6561 243 81 9 3 1

left pan: 4782969 59049 6561 243
right pan: 14348907 2187 729 9

left pan: 531441 19683 6561 2187 243 81 1
right pan: 1594323 177147 9

left pan: 177147 59049 243 9
right pan: 4782969 1594323 531441 81 27 1

left pan: 4782969 531441 59049 2187 27 9 3 1
right pan: 14348907 19683 6561 81

left pan: 4782969 1594323 531441 6561 2187 729 243 27 9 3 1
right pan: 14348907 19683

left pan: 531441 59049 6561 2187 27 1
right pan: 4782969 19683 243

left pan: 6561 243 81 1
right pan: 1594323 531441 177147 59049 19683 729 9

left pan: 531441 19683 81 3
right pan: 4782969 1594323 2187 729 243

left pan: 4782969 1594323 2187 729 27 9 3 1
right pan: 14348907 531441 6561 243 81

left pan: 729 81 3 1
right pan: 4782969 1594323 59049 19683 6561 2187 27

left pan: 4782969 1594323 177147 729 243 9 1
right pan: 14348907 531441 59049 19683

left pan: 4782969 1594323 531441 729 27 3 1
right pan: 14348907 19683 6561 2187 243 9

left pan: 1594323 2187 81 9 3
right pan: 4782969 19683 6561 243 1

left pan: 177147 27
right pan: 4782969 531441 59049 6561 2187 729 81 9
